testing.log

```diff
@@ -0,0 +1,6 @@
+## test of pipeline with addition of modycf profile 2022-02-02 ##
```

Comment:
Probably not intended to be committed?

Reply:
nope, what's the commit history of that? Going to be a hard purge

Reply:
I guess you can either make a new branch where you copy over all changes except this one, or use a more drastic tool:

https://stackoverflow.com/questions/43762338/how-to-remove-file-from-git-history

git-filter-repo seems to be what is recommended ..

Reply:
I would probably just do the copy approach here for such few changes

Reply:
@A97paupic dont commit lab-ids

Reply:
Aha, actually this was initially commited by you @ViktorHy in the commit 85b0d3979080 (just to put the record straight ;) )

Reply:
fair enough, I cannot argue with hard evidence. I apologize @A97paupic

Reply:
No worries! 😊